gems/sorbet-runtime/lib/types/props/custom_type.rb

```diff
@@ -80,5 +80,17 @@ def self.valid_serialization?(val, type=nil)
         scalar_type?(val)
       end
     end
+
+    def self.checked_serialize(type, instance)
```

Comment:
This is extracted from the old `T::Props::Serializable#serialize` implementation, so that we don't have to codegen it.